src/common/templates/includes/macros.njk

```diff
@@ -24,13 +71,11 @@
 {% macro submit_buttons(analyticsPage, csrfToken, __) %}
   <input id="save-and-continue" class="govuk-button" type="submit"
     name="buttonClicked"
-    value="{{__('form_save_and_continue')}}"
-    onclick="sendAnalytics(event, '{{ analyticsPage }} - Save and Continue', 'click')">
+    value="{{__('form_save_and_continue')}}">
   <p class="govuk-body-m">
     <input id="save-and-come-back" class="disableAutoFillSubmit govuk-button govuk-button--secondary" type="submit"
       name="buttonClicked"
-      value="{{__('form_save_and_come_back')}}"
-      onclick="sendAnalytics(event, '{{ analyticsPage }} - Save and Exit', 'click')">
```

Comment:
`sendAnalytics` when clicked will cause an error.
* This is because the `sendAnalytics` function is  `undefined` in app.
* It is not part of google analytics from googling it, so it is not something which needs to be configured.
* It was remove, when I was debugging why the form was submitting, in earlier development.


<img width="1440" alt="Screenshot 2024-04-03 at 12 52 38" src="https://github.com/UKHomeOffice/egar-public-site-ui/assets/123387957/298b16e1-ed17-4a39-b32b-c1b285b71d5a">

Reply:
I did not remove more of it to avoid going over scope.